tests/integration/test_config_changes.py

```
@@ -0,0 +1,787 @@
+            assert 'codecov.yaml' not in content, \
+                f"{workflow_file.name} should not reference codecov.yaml"
+            assert 'codecov.yml' not in content or 'codecov.yml' == workflow_file.name, \
+                f"{workflow_file.name} should not reference codecov.yml workflow"
+    
+    def test_coverage_still_collected(self):
+        """Test that coverage is still being collected after changes."""
+        with open('.github/workflows/ci.yml', 'r', encoding='utf-8') as f:
+            content = f.read()
+        
+        # Should still run pytest with coverage
+        assert 'pytest --cov' in content or 'pytest' in content and '--cov' in content, \
```

Ruff (0.14.6)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)